src/utils/helpers/json_templates.js

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
I think we need the copyright to be added if we modify this file. You could push the changes to the library's github. I think this is good functionality!

Reply:
Done, I will push it to library as well.

Reply:
Thanks @nxanil let's push it to master for now and then once the lib is updated we can use the package